Comp2024/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -294,9 +296,13 @@ public Command getAutonomousCommand( )
     {
       default :
       case AUTOSTOP :
+        m_autoCommand = new AutoStop(drivetrain);
```

Comment:
This switch statement seems a little consistent, since sometimes it selects a pathname and other times it selects a command. It could be more consistent.